torch/csrc/cuda/Module.cpp

```diff
@@ -979,6 +984,10 @@ static struct PyMethodDef _THCPModule_methods[] = {
      THCPModule_setBenchmarkLimitCuDNN,
      METH_O,
      nullptr},
+    {"_cuda_set_graphs_debug_path",
```

Comment:
I think instead of `_cuda_set_graphs_debug_path` you should have something like `enable_cudagraph_debug_mode` that will preserve `graph_` object for further inspection - you might want to do something other than dumping dot graph. As for path to dump the file, debug_dump itself should accept path argument - that would make it more consistent with other io functions.

Reply:
> As for path to dump the file, debug_dump itself should accept path argument - that would make it more consistent with other io functions.

:+1: as this was also my first approach while testing the PR without looking at the code